<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stencil</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User Application</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddUserBtn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>190</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stencil</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add User</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ViewUserBtn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stencil</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>View User</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="Title3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stencil</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User Application</string>
      </property>
     </widget>
     <widget class="QListWidget" name="UserList">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>541</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveBtn">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>100</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveAllBtn">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BackBtn3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>500</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>509</x>
        <y>500</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add User</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="Title2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Stencil</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User Application</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserNameLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="UserNameLine">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="EmailLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EmailLine">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BackBtn2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>360</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
